src/python/pants/backend/docker/goals/package_image.py

```diff
@@ -131,40 +139,55 @@ def image_refs(
         registries: DockerRegistries,
         interpolation_context: InterpolationContext,
         additional_tags: tuple[str, ...] = (),
-    ) -> tuple[str, ...]:
-        """The image refs are the full image name, including any registry and version tag.
+    ) -> Iterator[ImageRefRegistry]:
+        """The per-registry image refs: the full image name including any registry and version tag.
 
         In the Docker world, the term `tag` is used both for what we here prefer to call the image
         `ref`, as well as for the image version, or tag, that is at the end of the image name
         separated with a colon. By introducing the image `ref` we can retain the use of `tag` for
         the version part of the image name.
 
-        Returns all image refs to apply to the Docker image, on the form:
+        Returns all image refs to apply to the Docker image, accessible within the `full_name`
+        attribute of each element of the `tags` field:
```

Comment:
I don't really understand this sentence. I think I understand what it tries to say, but I fail to make the connection.